### PR TITLE
Make MultivariateTimeSeries::value_unit_ const

### PR DESCRIPTION
**Pull request by dimitry-**
Test: builds

### CODE REVIEW

File: src/OrbitGl/LineGraphTrack.h [hunk 1]
```diff
@@ -17,13 +17,7 @@ namespace orbit_gl {
 template <size_t Dimension>
 class LineGraphTrack : public GraphTrack<Dimension> {
  public:
-  explicit LineGraphTrack(CaptureViewElement* parent, TimeGraph* time_graph,
```
Comment (karupayun) [inline src/OrbitGl/LineGraphTrack.h:20]: Happy for this (and a bit surprised)

**Review by karupayun — APPROVED** [review]
LPTM! :)

**Review by vickyliu-go4it — APPROVED** [review]
That's really nice! Thanks for making this!

File: src/OrbitGl/LineGraphTrack.h (hunk 1 above)
Comment (dimitry-) [reply src/OrbitGl/LineGraphTrack.h:20]: Yep, I learned this trick from @beckerhe :)

**Pull request merged**